docs/eck-attributes.asciidoc

```diff
@@ -1,5 +1,5 @@
-:eck_version: 2.10.0
+:eck_version: 2.11.0
 :eck_crd_version: v1
-:eck_release_branch: 2.10
+:eck_release_branch: 2.11
 :eck_github: https://github.com/elastic/cloud-on-k8s
 :eck_resources_list: Elasticsearch, Kibana, APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash
```

Comment:
off topic: just realized that Stack Config Policy is missing in that list. But it's not an application and it would not always make sense, as in https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-custom-images.html:

> You can create your own custom application images (Elasticsearch, Kibana, APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash) instead of using the base images provided by Elastic